snake_cafe.py

```diff
@@ -79,42 +86,67 @@ def greeting():
 
     '''))
 
-question = dedent('''
-            **  Place order for another item  **\n
-            **  Type order to see your current order  **\n
-        ''')
+def start_order():
+    greeting()
+    item =  input('Would you like custom menu: ').lower()
```

Comment:
You should consider providing more explicit instructions here -- let the user know that they can load a custom menu by putting a .csv file in this directory, called "menu.csv", before they say yes to this prompt.